packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue

```diff
@@ -4,7 +4,7 @@
       <oc-button
         :id="`edit-public-link-role-dropdown-toggle-${link.id}`"
         appearance="raw"
-        class="edit-public-link-role-dropdown-toggle oc-text-left"
+        class="edit-public-link-role-dropdown-toggle oc-text-left oc-pill oc-background-default oc-px-s"
```

Comment:
Can you please remove `oc-pill` again? Looking at it again today I find it rather noisy. Sorry for the back and forth. I.e. only keep the pill style for the invite.

Reply:
The class definition or the whole applied pill style? pill style was proposed by @tbsbdr

Reply:
Using the oc-pill for the links. The css class definition is good and useful.